### PR TITLE
Update the 'Creating your first Tarantool database' guide

### LINKED ISSUE #3704: Update the 'Creating your first Tarantool database' guide
**Product:** Tarantool
**Since:** 3.0
**Root document:** https://www.tarantool.io/en/doc/latest/how-to/getting_started_db/
**SME:** @ Totktonada

# Details

The basic tutorial can use the interactive session for simplicity but the configuration file should be mentioned as the production approach (https://github.com/tarantool/doc/issues/3651). It's worth moving this tutorial to a new Getting Started section (to be created https://github.com/tarantool/doc/issues/3636). In the new Getting Started section, the interactive console might be the first way to try Tarantool. Then, we can show how to run a single instance using a config file, create a replica set, cluster, application, etc.

**UPD**: At this point, a new config looks better for a new Getting Started guide. Also, the tutorial should be updated to using tt.

### PR DESCRIPTION
**Pull request by xuniq**
Fixes #3704 

Staging: https://docs.d.tarantool.io/en/doc/gh-3704-create-db-guide/how-to/getting_started_db/

### CODE REVIEW

File: doc/how-to/getting_started_db.rst [hunk 1]
```diff
@@ -1,13 +1,282 @@
...
+Installing Tarantool
+--------------------
 
-.. include:: using_docker.rst
+For production purposes, we recommend that you install Tarantool via the
+`official package manager <http://tarantool.org/download.html>`_.
+You can choose one of three versions: LTS, stable, or beta.
+An automatic build system creates, tests, and publishes packages for every
+push into a corresponding branch at
+`Tarantool's GitHub repository <https://github.com/tarantool/tarantool>`_.
```
Comment (Totktonada) [inline doc/how-to/getting_started_db.rst]: This is not exactly so. We don't offer per-push builds via our repositories anymore.

Anyway, this is a technical detail and I would just remove this sentence instead of describing our deployment process. It looks irrelevant for this guide.

File: doc/how-to/getting_started_db.rst [hunk 2]
```diff
@@ -1,13 +1,282 @@
...
+create a database, and establish a remote connection using the :ref:`net.box <net_box-module>` module.
 
-You can install Tarantool and work with it locally or in Docker.
+Installing Tarantool
+--------------------
 
-.. include:: using_docker.rst
+For production purposes, we recommend that you install Tarantool via the
+`official package manager <http://tarantool.org/download.html>`_.
+You can choose one of three versions: LTS, stable, or beta.
```
Comment (Totktonada) [inline doc/how-to/getting_started_db.rst]: The link shows something different from these three options.

image

Is is OK?

File: doc/how-to/getting_started_db.rst [hunk 3]
```diff
@@ -1,13 +1,282 @@
...
+
+Connecting remotely
+-------------------
+
+In the configuration file (``config.yaml``), the instance listens on ``127.0.0.1:3301``:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/create_db/config.yaml
+    :language: yaml
+    :lines: 6-8
+    :dedent:
```
Comment (Totktonada) [inline doc/how-to/getting_started_db.rst]: It renders as so:

```yaml
instance001:
  iproto:
    listen:
```

File: doc/how-to/getting_started_db.rst [hunk 4]
```diff
@@ -1,13 +1,282 @@
...
+
+The ``listen`` value can be any form of a :ref:`URI <index-uri>` (uniform resource identifier).
+You can send requests to the listen URI using:
```
Comment (Totktonada) [inline doc/how-to/getting_started_db.rst]: It is not so now, see #3945.

File: doc/how-to/getting_started_db.rst [hunk 5]
```diff
@@ -1,13 +1,282 @@
...
+
+*   ``telnet``
+*   a :ref:`connector <index-box_connectors>`
+*   another instance of Tarantool (using the :ref:`console <console-module>` module)
+*   :ref:`tt <tt-cli>` administrative utility
```
Comment (Totktonada) [inline doc/how-to/getting_started_db.rst]: I guess it is for a control (console) socket. `iproto.listen` is about a binary protocol socket. You can't work with it using telnet.

OTOH, you can't use a connector to connect to a console (not a binary protocol socket).

Maybe we should describe it in a more general context, not just `iproto.listen`?

**Review by Totktonada — APPROVED** [review]
I'm OK in general.

Highlighted several chunks that doubt me a bit.

**Review by p7nov — APPROVED** [review]
@xuniq I took the liberty to somehow finalize this PR to be able to publish it with 3.0 docs – we don't have another starting point right now, so it's critical. 
You can return to this task later if you find it unfinished.
